File: Data Structures & Algorithms/car-fleet/submission-0.py
class Solution:
    def carFleet(self, target: int, position: List[int], speed: List[int]) -> int:
        cars = [(pos , (target - pos) /spd)for pos,spd in zip(position ,speed)]

        cars.sort(reverse = True)
        fleets = 0
        stack = []

        for pos,time in cars :
            if stack and time <=stack[-1]:
                continue
            else:
                stack.append(time)
        

        return len(stack)

File: Data Structures & Algorithms/reverse-string/submission-0.cpp
class Solution {
public:
    void reverseString(vector<char>& s) {


        int l = 0 ; 
        int r = s.size()-1;
        char a , b ;

        while(l<r){
            a = s[l];
            b = s[r];

            s[l] = b;
            s[r] = a;

            l++;
            r--;

        }
        
        
    }
};